nacc/uds3/fvp/builder.py

```diff
@@ -543,6 +543,15 @@ def build_uds3_fvp_form(record):
     b9.FTLDEVAL  = record['fu_ftldeval']
     packet.append(b9)
 
+    # Checking if those records are filled with 0 and marking as incomplete to those forms
+    if(record['fu_mmseloc'].strip() == 0):
```

Comment:
`s.strip()` returns a string. You shouldn't compare a string to an integer (`0`) like this.

Reply:
Thanks Taeber!